### ISSUE TITLE
How to handle 2nd browser popup window?

### ISSUE THREAD

**Issue by dtnyc9005** [adamdriscoll/selenium-powershell#65]
Hi guys, have an application that features a second (browser) window to pop up, and curious about how we would get into and manipulate it as we do with the parent window? any guidance would be appreciated...

-DT

**Comment by craigofnz** [comment]
@dtnyc9005 Can you identify the child window by iterating through Get-WindowHandle and Switch-SeWindow?

Would a helper function like [Select-SeWindow](https://github.com/craigofnz/selenium-powershell/blob/f69133d224200cf471b49cd4df0c50ba07013037/Selenium.psm1#L1474) help?

**Comment by dtnyc9005** [comment]
@craigofnz  Yes, that would be super helpful, like piping $driver | Select-SeWindow. Something like Select-SeWindow also knowing how many are currently open by the webdriver, will make things simpler that using the method SwitchTo().Window().
